apps/balancer-tools/src/app/apr/round/(components)/Breadcrumb.tsx

```diff
@@ -0,0 +1,128 @@
+"use client";
+import {
+  ChevronRightIcon,
+  ClipboardIcon,
+  ExternalLinkIcon,
+} from "@radix-ui/react-icons";
+import Link from "next/link";
+import React, { ReactNode } from "react";
+
+import { Pool } from "#/lib/balancer/gauges";
+
+import { Round } from "../../(utils)/rounds";
+
+function BreadcrumbItem({
+  content,
+  link,
+  children,
+}: {
+  content: string;
+  link?: string;
+  children?: ReactNode;
+}) {
+  return (
+    <li aria-current="page">
+      <div className="flex items-center">
+        <ChevronRightIcon />
+        {link ? (
+          <a
+            href={link}
+            className="flex ml-1 text-sm font-medium text-white hover:text-blue-600 md:ml-2"
+          >
+            {content}
+            {children}
```

Comment:
the way you created this component , both `children` and `content` does not add much

```js
          <BreadcrumbItem
            content={selectedPool?.symbol ?? poolId}
            link={`/apr/pool/${network}/${poolId}`}
          />
```

I would exclude one of these, or keep the `content` prop and exclude `children` and use the component as above or keep `children` and use the component like this:

```js
          <BreadcrumbItem
            link={`/apr/pool/${network}/${poolId}`}
          >
            {selectedPool?.symbol ?? poolId}
          </BreadcrumbItem>
```

Reply:
Agree, much cleaner this way!